client/src/utils/date.ts

```diff
@@ -0,0 +1,45 @@
+const filterDate = (today: Date) => {
+  const year = today.getFullYear();
+  const month = today.getMonth() + 1;
+  const date = today.getDate();
+  return [year, month as string | number, date as string | number];
```

Comment:
밑에서 문자열로 쓰기 위해서 여기서 as를 쓰는 것 보다 아래에서 새로운 변수에 할당하는게 더 좋은 것 같습니다.

Reply:
아래에서 새로운 변수에 할당을 하면 중복되는 함수에서 as를 다시 써줘야 하는데
그래도 아래에서 하는게 더 괜찬흥ㄹ까요?

Reply:
중복되는 함수에서 as를 써줘야한다는게 무슨말이신가요?

Reply:
네 아니면 중복되는 
```suggestion
  const month = today.getMonth() + 1 as string | number;
  const date = today.getDate() as string | number;
  return [year, month, date];
```
이렇게 해주는 것이 좀 더 가독성있지 않을까요?

사실 as를 안 쓰는 쪽이 더 좋은 것 같아요!

Reply:
변경완료(페어프로그래밍)